de.metas.endcustomer.mf15/de.metas.endcustomer.mf15.swingui/pom.xml

```diff
@@ -120,7 +120,27 @@
 			<version>${metasfresh.version}</version>
 			<scope>provided</scope>
 		</dependency>
-
+		
```

Comment:
Hi @homebeaver ,
for us to integrate this PR, can you please provide a PR without this changed `pom.xml`? It that's too much effort, please let us know. Then we can manually integrate the rest without this.